fw/cfg.h

```diff
@@ -201,14 +201,21 @@ typedef struct {
 		return -EINVAL;						\
 	}
 
-#define TFW_CFG_CHECK_VAL_N(op, req, spec, entry)				\
+#define TFW_CFG_CHECK_VAL_N(op, req, spec, entry)			\
 	if (! ((entry)->val_n op (req)) ) {				\
 		T_ERR_NL("%s: Invalid number of arguments: %zu, must "	\
 			 "be %s %d\n", (spec)->name, (entry)->val_n,	\
 			 #op, (req));					\
 		return -EINVAL;						\
 	}
 
+#define TFW_CFG_CHECK_VAL_DUBLICATE(name, val_was_set, code)		\
```

Comment:
While spelling `dublicate` exists, it's very rare. `Duplicate` is the right word. Also, perhaps, the standard shortcut would suffice here for a shorter name? `TFW_CFG_CHECK_VAL_DUP`.

Reply:
Yes fixed